index.html

```diff
@@ -1325,6 +1327,44 @@ <h2>
 detail how revocation is performed and tracked.
       </p>
 
+      <p>
+All public key formats MUST be expressed in either JSON Web Key (JWK)
+format using the <code>publicKeyJwk</code> property or one of the native
+formats listed in the table below. All other key formats MUST NOT be used.
+      </p>
+
+      <p class="issue">
+The Working Group is still debating whether the default base encoding format
```

Comment:
Change "default base encoding format" to "base encoding format used".  (There are no defaults - only specified representations.)